.kokoro/samples/python3.11/common.cfg

```diff
@@ -0,0 +1,40 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
@tswast 

are these files typically autogenerated OR do we add the manually?

Reply:
I suspect that these are autogenerated. However, I tried adding them manually as we had a test failure relating to Kokoro: https://source.cloud.google.com/results/invocations/b41e3d1e-1e73-454d-a24a-d1a790c7b129/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-bigquery-sqlalchemy%2Fpresubmit%2Fpresubmit/log
I believe that the failure was the result of `Session unit-3.11 failed: Python interpreter 3.11 not found.` and I was experimenting whether adding these files resolved that issue. If it doesn't I'll go ahead and remove those files.

Reply:
Oh shoot - it appears that I can't find that out the results of that experiment without adding the `kokoro:run`, which I'm unable to do. I'll revert those additions as it was a slim chance that this would fix the build.